### PR TITLE
Tanvi

### PR DESCRIPTION
**Pull request by TanviParsekar**
Done the changes as discussed.
Used radio, text, password, and button type input.
I have left the link part empty. So do check that out and let me know if I need to change anything else in the code.

### CODE REVIEW

File: src/login_page/login_page.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	  <img src="../global/assets/logo_with_name_initials.svg" height="45px" id="header-logo">
+	</picture>
+      </a>
+    </header>
+    
+    <!-- Content -->
+    <main>
+      <form id="login-form" action="TODO" method="post">
+        <ul>
+        <li>
```
Comment (Shriteen) [inline src/login_page/login_page.html:28]: There should be more indentation for contents of ul

File: src/login_page/login_page.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          <input type="radio" id="login-account-type-employer" name="account-type">
+          <label for="login-account-type-employer">Find Work</label>
+	  <input type="radio" id="login-account-type-freelancer" name="account-type">
+	  <label for="login-account-type-freelancer">Find Talent</label>
+        </li>
+
+        <li>
+          <!-- Username -->
+	  <label for="username">Username</label>
+	  <input type="text" id="username" name="account-name" maxlength="100">
```
Comment (Shriteen) [inline src/login_page/login_page.html:38]: Change the id to login-account-username for consistency (also update the for attribute in label)
Do same for password -> login-account-password

File: src/login_page/login_page.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    
+        <li>
+          <!-- Password -->
+	  <label for="password">Password</label>
+	  <input type="password" id="password" name="account-password" maxlength="100">
+        </li>
+
+        <li>
+          <button type="button" id="login-button">Log In</button>
+        </li>
```
Comment (Shriteen) [inline src/login_page/login_page.html:49]: Change button type to submit

File: src/login_page/login_page.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+        Don't have an account? <a href="">Sign Up</a>
+        </ul>
```
Comment (Shriteen) [inline src/login_page/login_page.html:52]: Add href="../signup_page/signup_page.html"
Also make whole line as link

File: src/login_page/login_page.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+	<picture alt="logo">
+	  <source media="(max-width:600px)" srcset="../global/assets/logo_with_name_initials.svg">
+	  <source media="(min-width:601px)" srcset="../global/assets/logo_with_name.svg">
+	  <img src="../global/assets/logo_with_name_initials.svg" height="45px" id="header-logo">
+	</picture>
+      </a>
+    </header>
+    
+    <!-- Content -->
+    <main>
```
Comment (Shriteen) [inline src/login_page/login_page.html:25]: Put required attribute for all inputs. Check the updated signup_page.html for reference

**Review by Shriteen — CHANGES_REQUESTED** [review]
Make the changes and push to Tanvi branch.
Pull request should automatically get updated I think.

File: src/login_page/login_page.html (hunk 4 above)
Comment (TanviParsekar) [reply src/login_page/login_page.html:52]: whole link as link meaning what exactly I have to do?
And I have added the href part.

**Review by Shriteen — APPROVED** [review]

**Pull request merged**